app/gui/src/project-view/components/visualizations/TableVisualization.vue

```diff
@@ -356,23 +356,20 @@ function toField(
   const displayValue = valueType ? valueType.display_text : null
   const icon = valueType ? getValueTypeIcon(valueType.constructor) : null
 
-  const dataQuality =
-    typeof props.data === 'object' && 'data_quality_pairs' in props.data ?
-      props.data.data_quality_pairs
-      // eslint-disable-next-line camelcase
-    : { number_of_nothing: [], number_of_whitespace: [] }
+  console.log(props.data)
 
-  const nothingIsNonZero =
-    index != null && dataQuality?.number_of_nothing ?
-      (dataQuality.number_of_nothing[index] ?? 0) > 0
-    : false
+  const dataQualityMetrics =
+    typeof props.data === 'object' && 'data_quality_metrics' in props.data ?
+      props.data.data_quality_metrics.map((metric: DataQualityMetric) => {
+        console.log({ metric })
+        return { [metric.name]: metric.percentage_value[index!] ?? 0 }
+      })
+    : []
```

Comment:
Log an error in this case as well? (Assuming it is a malformed data error...)

Reply:
It's not an error just no metrics sent.